build/ci.go

```diff
@@ -173,6 +173,21 @@ func doInstall(cmdline []string) {
 	if flag.NArg() > 0 {
 		packages = flag.Args()
 	}
+
+	if runtime.GOOS == "windows" {
+		// Resolve ./... manually and remove vendor/bazil/fuse (fuse is not in windows)
+		out, err := goTool("list", "./...").CombinedOutput()
+		if err != nil {
+			log.Fatalf("package listing failed: %v\n%s", err, string(out))
+		}
+		packages = []string{}
+		for _, line := range strings.Split(string(out), "\n") {
+			if !strings.Contains(line, "bazil") {
+				packages = append(packages, strings.TrimSpace(line))
+			}
+		}
+	}
```

Comment:
It cannot be removed for all platforms, as some fuse test are dependent on it.
It is removed for windows since windows does not support fuse or user level file system support  in general.